app/event_handlers/response_pdf_file_handler.rb

```diff
@@ -1,6 +1,6 @@
 class ResponsePdfFileHandler
   def handle(response)
-    ResponseFileBuilderService.new(response).call
+    BuildResponsePdfFileService.call(response, template_reference: response.pdf_template_reference)
```

Comment:
A  bit of refactoring here.  The response pdf file handler used to be part of a bigger 'Response File Builder' a while ago - and their were still parts of this code left even though the response file builder now only builds the pdf file.
This is because the other files were all atos files which were extracted into the atos gem